config/crd/bases/actions.summerwind.dev_horizontalrunnerautoscalers.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
+    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
nit: You'd better use a newer version of kubebuilder (0.3.0)

Reply:
I wasn't sure what the issue was here, but Ive updated the version and now it seems to be fine